Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -13,11 +13,19 @@ public static class ResultManager
     {
         private static PluginInitContext Context;
         private static Settings Settings { get; set; }
+        public static object Keyword { get; private set; }
 
         public static void Init(PluginInitContext context, Settings settings)
         {
             Context = context;
             Settings = settings;
+            Keyword = Settings.SearchActionKeywordEnabled ? Settings.SearchActionKeyword : Settings.PathSearchActionKeyword;
+            Keyword = Keyword.ToString() == Query.GlobalPluginWildcardSign ? string.Empty : Keyword + " ";
```

Comment:
i see why you did it this way, but because the action keyword could change during the runtime of the app, this means Keyword is only set once during Init. We need to get this dynamically because could change anytime.

Reply:
i am working on tweaking this, will push up when done.

Reply:
In my testing it worked even when hotkey changed.

Reply:
done, please test it now